cartographer/cloud/client/map_builder_stub.cc

```diff
@@ -77,6 +78,7 @@ int MapBuilderStub::AddTrajectoryForDeserialization(
     const mapping::proto::TrajectoryBuilderOptionsWithSensorIds&
         options_with_sensor_ids_proto) {
   LOG(FATAL) << "Not implemented";
+  return -1;
```

Comment:
This should compile fine w/o the return statement.

Reply:
I did get a compilation error at some point, that's why I added it.

Reply:
removed.